modal_proto/api.proto

```diff
@@ -996,7 +996,7 @@ message GPUConfig {
 }
 
 message Resources {
-  bool gpu = 1; // Legacy GPU toggle.
+  reserved 1; // Legacy boolean "gpu" field
```

Comment:
you can remove this line, i think – it's implied that the field has been removed unless it's the last one